spec/dummy/db/schema.rb

```diff
@@ -37,6 +37,10 @@
     t.datetime "avatar_updated_at"
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.string   "email"
+    t.string   "twitter"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.